modules/installation-deploymentmanager-security.adoc

```diff
@@ -0,0 +1,153 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp_user_infra/installing-gcp-user-infra.adoc
+
+[id="installation-deployment-manager-security_{context}"]
```

Comment:
This file seems to be a copy of `modules/installation-deployment-manager-security.adoc`